src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -201,15 +196,6 @@ class QueueIO[T <: Data](private val gen: T, val entries: Int) extends Bundle
     * @group Signals
     */
   val count = Output(UInt(log2Ceil(entries + 1).W))
-<<<<<<< HEAD
-=======
-  /** When asserted, reset the enqueue and dequeue pointers, effectively flushing the queue (Optional IO for a flushable Queue)
-    * @group Signals
-    */ 
-  val flush = if (hasFlush) Some(Input(Bool())) else None
-
->>>>>>> a79f5756 (Add field grouping ScalaDoc for other subclasses of Bundle  (#2214))
-}
```

Comment:
you probably still want this `}`

Reply:
Thank you @mwachs5, I put it back there.